blog/crsp-v2-update/index.qmd

```diff
@@ -0,0 +1,133 @@
+
+You can read all about how to connect to WRDS and download the CRSP data in our chapter **WRDS, CRSP, and Compustat** in the [R version](../../r/wrds-crsp-and-compustat.qmd) and the [Python version](../../python/wrds-crsp-and-compustat.qmd).
+
+## CRSP format 2.0 (CIZ)
+
+In 2022, CRSP rolled out the new format of its data which is called **CRSP's Stock and Indexes Flat File Format 2.0 (CIZ)**. This new format is now also available via WRDS. So, what is new in the second version? The most notable items from CRSP's announcement includes:
+
+- They renamed many variables in the daily and monthly data. Not all of these changes make it easier to understand what the variables are, and some names are hard to read. 
+- They added additional variables that are sometimes redundant but make querying simpler (e.g., you do not have to merge different data to perform common tasks).
+- Delisting returns are now included in the main return time series.
+- They introduce issuer level data, which involved ensuring unique observations for each issuer (e.g., industry codes).
+- They reworked some flag items by giving them alphanumeric values instead of the previous numeric codes. Moreover, these codes are now also documented in metadata files to facilitate access.
+
+Many of these changes mean, that you will not be able to simply use the new data with your existing code. Do not worry. We have you covered and will show you how to use the new data in your previous routines easily.
+
+You can access the new data via WRDS using the commands below. For the main data WRDS basically just added a "_v2" postfix. However, you also see that we need fewer tables, as we do not load information on delisting returns separately. 
+
+::: {.panel-tabset group="language"}
+## R
+
+```{r}
+#| eval: false
+msf_db <- tbl(wrds, in_schema("crsp", "msf_v2"))
+stksecurityinfohist_db <- tbl(wrds, in_schema("crsp", "stksecurityinfohist"))
+```
+
+## Python
+
+```{python, eval = FALSE, python.reticulate = FALSE}
+#| eval: false
+crsp_monthly_query = (
+  "SELECT msf.permno"
+    "FROM crsp.msf_v2 AS msf "
+    "LEFT JOIN crsp.stksecurityinfohist AS ssih "
+  )
+```
+:::
+
+We do not repost the new routines for downloading the data in this blog. Instead, we ask you to check out the new code in the respective chapter on **WRDS, CRSP, and Compustat** in the [R version](../../r/wrds-crsp-and-compustat.qmd) and the [Python version](../../python/wrds-crsp-and-compustat.qmd).
+
+## A small glimpse at the differences
+
+To take a look at the new table and compare it to the old, we use the new `tidyfinance` R package to download the data. If you do not have the package yet, please install it from CRAN using `install.packages("tidyfinance")`. For ease of use, we also load the full `tidyverse` as in all our chapters of the R book.
```

Comment:
Add a link to the r-tidyfinance-0-1-0 post in parentheses e.g. (see our blog post on the initial release [here](../r-tidyfinance-0-1-0/index.qmd).